db/schema.rb

```diff
@@ -10,119 +10,120 @@
+    t.check_constraint "evaluation = ANY (ARRAY[1, 2, 3, 4, 5])", name: "evaluation_check_constraint"
   end
 
-  create_table 'resources', force: :cascade do |t|
-    t.string 'name'
-    t.string 'url'
-    t.datetime 'created_at', null: false
-    t.datetime 'updated_at', null: false
-    t.bigint 'user_id'
-    t.bigint 'learning_unit_id'
-    t.index ['learning_unit_id'], name: 'index_resources_on_learning_unit_id'
-    t.index ['user_id'], name: 'index_resources_on_user_id'
+  create_table "resources", force: :cascade do |t|
+    t.string "name"
+    t.string "url"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "user_id"
+    t.bigint "learning_unit_id"
+    t.index ["learning_unit_id"], name: "index_resources_on_learning_unit_id"
+    t.index ["user_id"], name: "index_resources_on_user_id"
   end
 
-  create_table 'users', force: :cascade do |t|
-    t.string 'name'
-    t.datetime 'created_at', null: false
-    t.datetime 'updated_at', null: false
-    t.string 'email'
-    t.string 'encrypted_password', default: '', null: false
-    t.string 'reset_password_token'
-    t.datetime 'reset_password_sent_at'
-    t.datetime 'remember_created_at'
-    t.index ['email'], name: 'index_users_on_email', unique: true
-    t.index ['reset_password_token'], name: 'index_users_on_reset_password_token', unique: true
+  create_table "users", force: :cascade do |t|
+    t.string "name"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.string "email"
+    t.string "encrypted_password", default: "", null: false
+    t.string "reset_password_token"
+    t.datetime "reset_password_sent_at"
+    t.datetime "remember_created_at"
+    t.index ["email"], name: "index_users_on_email", unique: true
+    t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
   end
 
-  add_foreign_key 'completed_learning_units', 'learning_units'
-  add_foreign_key 'completed_learning_units', 'users'
-  add_foreign_key 'curriculum_affiliations', 'curriculums'
-  add_foreign_key 'curriculum_affiliations', 'learning_units'
-  add_foreign_key 'resource_comments', 'resources'
-  add_foreign_key 'resource_comments', 'users'
-  add_foreign_key 'resource_evaluations', 'resources'
-  add_foreign_key 'resource_evaluations', 'users'
-  add_foreign_key 'resources', 'learning_units'
-  add_foreign_key 'resources', 'users'
+  add_foreign_key "completed_learning_units", "learning_units"
+  add_foreign_key "completed_learning_units", "users"
+  add_foreign_key "curriculum_affiliations", "curriculums"
+  add_foreign_key "curriculum_affiliations", "learning_units"
+  add_foreign_key "resource_comments", "resources"
+  add_foreign_key "resource_comments", "users"
+  add_foreign_key "resource_evaluations", "resources"
+  add_foreign_key "resource_evaluations", "users"
+  add_foreign_key "resources", "learning_units"
+  add_foreign_key "resources", "users"
```

Comment:
Not that important, but in a previous PR comment @rtichauerv suggested to use single quotes instead of double quotes to maintain uniformity.